desktop/cmp/rest/impl/RestControlModel.js

```diff
@@ -39,8 +39,8 @@ export class RestControlModel  {
 
     @computed
     get value()  {
```

Comment:
If we are going to handle this here, would like to generalize this from handleBoolean to parseStringValue.  why are we handling just "false" => false? Why not "true" to true?  or "3" to 3.?